.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -272,10 +276,10 @@ jobs:
             cargo_dir: ~/.cargo
```

Comment:
clang-format on the previous line changed it's patch to `/Users/runner/Library/Python/3.13/bin/clang-format` for macOS 13